.lintstagedrc.js

```diff
@@ -6,5 +6,5 @@ const buildEslintCommand = (filenames) =>
 		.join(' --file ')}`
 
 module.exports = {
-	'*.{js,jsx,ts,tsx}': [buildEslintCommand],
```

Comment:
Removed `.js(x)` from linting due to some inexplicable behavior when linting firebase/seed; in practice, we should stick to `.ts(x)` strictly within the Next.js client app, anyways (firebase/seed is mostly just for reference, its main purpose is to initialize the prod data, which has already been done, but otherwise not part of the client app itself)